phpstan-baseline.neon

```diff
@@ -30,6 +30,16 @@ parameters:
 			count: 1
 			path: src/Transport/Guzzle.php
 
+		-
```

Comment:
Trying to get my head around why we don't fix these in the test files directly? Related to breaking changes?

Reply:
This is what it was explained in the PR description https://github.com/ruflin/Elastica/pull/2108#issue-1328708352, a test like this: 

https://github.com/ruflin/Elastica/blob/8fe61767b604a10e40b0b59c9511c4317cae3cb8/tests/Aggregation/GeohashGridTest.php#L51-L57

makes PHPStan fails because `setPrecision` only accepts `int|string`:

https://github.com/ruflin/Elastica/blob/6d51cdbff2fb37aceac8b70adf9042ccb9be0aed/src/Aggregation/GeohashGrid.php#L42-L50

We can get rid of these ignored errors and checks in the next major if we add type parameter declarations.